include/benchmark/benchmark_api.h

```diff
@@ -574,6 +574,8 @@ class Benchmark {
   //    Foo in 16 threads
   Benchmark* ThreadRange(int min_threads, int max_threads);
 
+  Benchmark* ThreadRange2(int min_threads, int max_threads);
```

Comment:
I would suggest `DenseThreadRange(int min_threads, int max_threads, int stride=1)` since it matches the `DenseRange` function we already have.